Components/Queues/QueueProducer.php

```diff
@@ -21,6 +22,7 @@ class QueueProducer extends Producer
 
     protected $headersToPropagate = [
         Message::HEADER_EXPIRES,
+        'x-dead-letters-exchange' => 'EAI'
```

Comment:
Is this header needed? And if it is which is the purpose

Reply:
No, it was just for test purposes. This line was removed now.